packages/react-jss/src/createHoc.js

```diff
@@ -109,6 +107,8 @@ export default function createHOC<
     constructor(props: OuterPropsType, context: Context) {
       super(props, context)
 
+      this.context = context
```

Comment:
@kof This fixes the weird issue with `react-jss` in the alpha.

Reply:
strange, parent class should have done this

Reply:
oh mb we are using context before super does the job